lib/ui/painting/image_decoder_impeller.cc

```diff
@@ -49,7 +49,9 @@ float CalculateArea(SkPoint abc[3]) {
                       b.fX * a.fY);
 }
 
-static constexpr float kSrgbD50GamutArea = 0.084f;
+// Note: This is defined differently in Skia, in my testing this is the correct
+// area and was calculated from SkColorSpace::MakeSRGB().
```

Comment:
nit: if Skia is updating this, consider linking the skbug for it and mentioning that here.

Reply:
done